### PR TITLE
Allow dependabot to write elixir unit test report

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes failing dependabot PRs

### CODE REVIEW

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -46,7 +48,7 @@ jobs:
       - name: Install Dependencies
         if: ${{ steps.cache.outputs.cache-hit != 'true' }}
         run: mix deps.get --only $MIX_ENV
-      - name: Compile Dependencies
+      - name: Compile Dependenciee
```
Comment (AndrewDryga) [inline .github/workflows/elixir.yml]: Est-ce français?

File: .github/workflows/elixir.yml [hunk 2]
```diff
@@ -8,6 +8,8 @@ jobs:
     defaults:
       run:
         working-directory: ./elixir
+    permissions:
+      checks: write
```
Comment (AndrewDryga) [inline .github/workflows/elixir.yml:12]: Will it also grant other permissions for not whitelisted contributors?

Comment (jamilbk) [reply .github/workflows/elixir.yml:12]: No, checks won't run for forks of our repo, so we should be good there.

File: .github/workflows/elixir.yml (hunk 1 above)
Comment (jamilbk) [reply .github/workflows/elixir.yml]: whoops, fixed!

File: .github/workflows/elixir.yml (hunk 2 above)
Comment (jamilbk) [reply .github/workflows/elixir.yml:12]: This is the same fix I applied in fa54a03b76 -- just needed here now too.

**Review by AndrewDryga — APPROVED** [review]

**Pull request merged**